cachi2/core/package_managers/bundler/main.py

```diff
@@ -14,18 +16,23 @@
 
 log = logging.getLogger(__name__)
 
+CONFIG_OVERRIDE = "bundle/config_override"
```

Comment:
s/bundle/bundler   - _bundle_ is the location Bundler places its stuff into according to the docs, _bundler_ is the namespace for our backend

Since it appears you've received more review comments and you'll be re-spinning one more time you might as well drop the additional directory level for the config override and just have the override config file in there directly.

Reply:
Done. I'd rather kept a directory -- its name contains a clue to what to expect inside.